sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileClient.java

```diff
@@ -848,28 +848,66 @@ public void uploadFromFile(String uploadFilePath) {
      *
      * <p>For more information, see the
      * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles">Azure Docs</a>.</p>
      *
-     * @param handleId Specifies the handle ID to be closed. Use an asterisk ('*') as a wildcard string to specify all
-     * handles.
+     * @param handleId Handle ID to be closed.
+     */
+    public void forceCloseHandle(String handleId) {
+        forceCloseHandleWithResponse(handleId, null, Context.NONE);
+    }
+
+    /**
+     * Closes a handle on the file at the service. This is intended to be used alongside {@link #listHandles()}.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Force close handles returned by list handles.</p>
+     *
+     * {@codesnippet com.azure.storage.file.FileClient.forceCloseHandleWithResponse#String}
+     *
+     * <p>For more information, see the
+     * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles">Azure Docs</a>.</p>
+     *
+     * @param handleId Handle ID to be clsoed.
      * @param timeout An optional timeout applied to the operation. If a response is not returned before the timeout
      * concludes a {@link RuntimeException} will be thrown.
      * @param context Additional context that is passed through the Http pipeline during the service call.
-     * @return The counts of number of handles closed
-     * @throws RuntimeException if the operation doesn't complete before the timeout concludes.
+     * @return A response that only contains headers and response status code.
+     */
+    public Response<Void> forceCloseHandleWithResponse(String handleId, Duration timeout, Context context) {
+        return Utility.blockWithOptionalTimeout(fileAsyncClient
+            .forceCloseHandleWithResponse(handleId, context), timeout);
+    }
+
+    /**
+     * Closes all handles opened on the file at the service.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Force close all handles.</p>
+     *
+     * {@codesnippet com.azure.storage.file.FileClient.forceCloseAllHandles#Duration-Context}
+     *
+     * <p>For more information, see the
+     * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles">Azure Docs</a>.</p>
+     *
+     * @param timeout An optional timeout applied to the operation. If a response is not returned before the timeout
+     * concludes a {@link RuntimeException} will be thrown.
+     * @param context Additional context that is passed through the Http pipeline during the service call.
+     * @return The number of handles closed.
      */
-    public PagedIterable<Integer> forceCloseHandles(String handleId, Duration timeout, Context context) {
-        return new PagedIterable<>(fileAsyncClient.forceCloseHandlesWithOptionalTimeout(handleId, timeout, context));
+    public int forceCloseAllHandles(Duration timeout, Context context) {
```

Comment:
Same here - should there be a `withResponse`?

Reply:
Direction for this API is to just return the number of handles closed as we are abstracting away the pagedness of the REST API.